components/atom/upload/src/index.scss

```diff
@@ -0,0 +1,61 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bg-atom-upload: #f5f5f5 !default;
+$c-atom-upload: #979797 !default;
+
+.sui-AtomUpload {
+  align-items: center;
+  background-color: $bg-atom-upload;
+  border: 2px dashed $c-atom-upload;
```

Comment:
please use sui-theme variables for this kind of stuff https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss#L5

Reply:
as no borders are defined yet we'll do it this way but create a jira task in order to solve it as soon as they are defined